compose/link-env-deprecated.md

```diff
@@ -39,5 +39,7 @@ Fully qualified container name, such as `DB_1_NAME=/myapp_web_1/myapp_db_1`
 
```

Comment:
This page isn't linked in the TOC - https://deploy-preview-10717--docsdocker.netlify.app/compose/link-env-deprecated/

Reply:
Good catch; should we remove the file and redirect it somewhere?

Reply:
I think that's fair. How about redirect to the Environment variables topic - https://deploy-preview-10717--docsdocker.netlify.app/compose/environment-variables/?